plugins/deployment_manager/MigrationTemplate.ts

```diff
@@ -12,12 +12,16 @@ import { migration } from '../../../../plugins/deployment_manager/Migration';
 interface Vars {};
 
 export default migration('${timestamp}_${name}', {
-  prepare: async (deploymentManager: DeploymentManager) => {
+  async prepare(deploymentManager: DeploymentManager) {
```

Comment:
just curious: why change this function from `fnName: async() =>` to `async fnName() {`?

Reply:
I noticed a lot of our migration scripts were using `async fnName()` so I changed this to be more consistent. But no preference on my end, so I'm down to revert this change if you want.